### DISCUSSION TITLE
[Resolved] Builds stuck in queue

### DISCUSSION THREAD

**Discussion by ahuggins** [vercel/community#2189] [Bug Report]
Been probably 20 minutes with my build saying "Queued" usually builds in 4 minutes. Anyone know what is up?

**Comment by WendyKKelly** [comment]
I've got the same issue - it's been over 16 minutes.

**Comment by sdornan** [comment]
Same here

**Comment by bottarocarlo** [comment]
Same issue here, though was my problem - over 30 minutes

**Comment by jspencersharpe** [comment]
Same issue

**Comment by guictrl** [comment]
Yep, me too

**Comment by mrpharderwijk** [comment]
Same issue

**Comment by PaulieScanlon** [comment]
same here

**Comment by MattWIP** [comment]
Same!

**Comment by ahuggins** [comment]
My builds seem to be going now.

**Reply by guictrl** [reply]
Did you have to cancel & redeploy?

**Reply by ahuggins** [reply]
I did, but it didn't seem to make a difference. Wonder if there was an issue with one build that finally timed out.

**Comment by swarnava** [comment]
Hi, apologies for the inconvenience caused. We are investigating this currently and share more updates here: https://www.vercel-status.com/incidents/z01svsnlrhcz

**Comment by swarnava** [comment]
This should be resolved now!
